tests/unit/h/views/admin/groups_test.py

```diff
@@ -316,6 +343,35 @@ def call_on_success(  # pylint:disable=unused-argument
         )
         assert response["form"] == self._expected_form(group)
 
+    def test_update_when_group_has_no_organization(
+        self, pyramid_request, handle_form_submission, group_update_service, group
+    ):
+        group.organization = None
+
+        def call_on_success(  # pylint:disable=unused-argument
+            request, form, on_success, on_failure
+        ):
+            return on_success(
+                {
+                    # No "organization" in these on_success() kwargs.
```

Comment:
There _is_ an "organization" field, but it is None. Is that what you meant?

Reply:
Fixed the comment